libraries/Microsoft.Bot.Builder.LUIS/ILuisRecognizer.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Cognitive.LUIS.Models;
+
+namespace Microsoft.Bot.Builder.LUIS
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// A Luis specific interface that extends the generic Recognizer interface
+    /// </summary>
+    internal interface ILuisRecognizer : IRecognizer
```

Comment:
There are no specific tests around this interface. Please implement a Unit Tests that implements this and makes sure the right behavior is present.

Reply:
This interface has ~100% coverage from previous change (via testing LuisRecognizer which now implements it).